jobs/build/pre-release/Jenkinsfile

```
@@ -0,0 +1,107 @@
+#!/usr/bin/env groovy
+
+node {
+    checkout scm
+    def prerelease = load("../release/release.groovy")
```

This would not actually work, it would need to be
```suggestion
    def prerelease = load("jobs/build/release/release.groovy")
```
... but let's not do that either, that's too coupled. Can you pull release.groovy into pipeline-scripts instead?

---

done